AppiumLibrary/keywords/_touch.py

```diff
@@ -93,6 +94,192 @@ def scroll(self, start_locator, end_locator):
         driver = self._current_application()
         driver.scroll(el1, el2)
 
+    def _generate_xpath_for_uiautomator(self, xpathtext):
+        print ">>> Generating XPath for UIAutomator... "
+
+        # 1. get class name
+        divider = xpathtext.find('[')
+        className = xpathtext[:divider]
+        className = className.replace('//','')
+
+        # 2. get property name
+        divider = xpathtext.find('@')
+        propName = xpathtext[divider+1:]
+        divider = propName.find(',')
+        propName = propName[:divider]
+
+        # 3. get property value
+        divider = xpathtext.find("'")
+        propValue = xpathtext[divider+1:]
+        divider = propValue.find("'")
+        propValue = propValue[:divider]
+
+        print ">>> Generating XPath for UIAutomator completed."
+        return className, propName, propValue 
+
+    def swipe_until_element_visible(self, x_start, y_start, x_end, y_end, locator, scroll_limit=50):
+        """
+        Swipes the device until element visible/found.
+
+        The default scroll limit before the scroll stops is 50. 
+        To override this, use "scroll_limit" argument at the end of keyword issuing
+
+
+        Author: thekucays

```

Comment:
Remove this line please, 
also is this android only, can you mention it the same way library has done before